updating/updating-cluster.adoc

```diff
@@ -15,6 +15,8 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * Ensure that all machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
 * If your cluster uses manually maintained credentials, ensure that the Cloud Credential Operator (CCO) is in an upgradeable state. For more information, see _Upgrading clusters with manually maintained credentials_ for xref:../installing/installing_aws/manually-creating-iam.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-aws[AWS], xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-azure[Azure], or xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-maintained-credentials-upgrade_manually-creating-iam-gcp[GCP].
 * If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being upgraded to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[_Upgrading an OpenShift Container Platform cluster configured for manual mode with STS_].
+* Ensure you address all Upgradeable=False conditions so the cluster will allow an upgrade to the next minor version. An alert will display at the top of the *Cluster Settings* page when you have one or more ClusterOperators that can not be upgraded. You will still be able to upgrade to the next available patch update for the minor release you are currently on. 
```

Comment:
Same comments from the above file (where applicable), plus:

* An alert will display -> An alert displays
* ClusterOperators -> cluster Operators
* can not -> cannot
* Reword "You will still be able to" to get rid of the future tense. Maybe "You can still"?